ROADMAP.md

```diff
@@ -1,39 +1,42 @@
 # Roadmap
-This document represents the current working roadmap for Pantheon.  It is a living document, which will evolve and change over time.  In particular the features in later versions are likely to be refined and change.
+This document represents the current working roadmap for Pantheon.  It is a living document, which will 
+evolve and change over time.  In particular the features in later versions are likely to be refined and change.
 
-We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  Our now scale is about 3 months, next about 6 months, and then later is beyond.
+We use the approach of  `#now`, `#next`, `#later` [used by foursquare](https://medium.com/@noah_weiss/now-next-later-roadmaps-without-the-drudgery-1cfe65656645), with a slightly different time horizon.  
+Our now scale is about 3 months, next about 6 months, and then later is beyond.
 
-## Now
-Our key three areas for now are:
-* Permissioning
+## Now/v1.1
```

Comment:
Nit, but perhaps `Now (up to v1.1)`, thoughts?

Reply:
Makes sense. Updated